### PR TITLE
Add quickstart section

### PR DESCRIPTION
**Pull request by evilnick**

### CODE REVIEW

**Comment by evilnick** [comment]
@ktsakalozos Apologies that most of this is html. It is the only way to do the stepped list layout which is important for several reasons...

File: docs/index.md [hunk 1]
```diff
@@ -1,227 +1,151 @@
...
+      </li>
+
+      <li class="p-stepped-list__item">
+        <h3 class="p-stepped-list__title"><span class="p-stepped-list__bullet">4</span>Deploy an app</h3>
+        <div class="p-stepped-list__content">
+          <p>Of course, Kubernetes is meant for deploying apps and services. You can use
+          the <code>kubectl</code> command to do that as with any Kuberenetes. Try
+          installing a demo app:</p>
+          <div class="p-code-copyable">
+            <input class="p-code-copyable__input" value="microk8s.kubectl create deployment kubernetes-bootcamp --image=gcr.io/google-samples/kubernetes-bootcamp:v1" readonly="readonly">
```
Comment (evilnick) [inline docs/index.md]: @ktsakalozos You may have a better idea for a demo?

**Comment by evilnick** [comment]
@ant - the p-stepped-list layout is great, but the equal columns mean some of the commands etc are truncated - know any quick fixes?

image

File: docs/index.md (hunk 1 above)
Comment (ktsakalozos) [reply docs/index.md]: There is --image=dontrebootme/microbot:v1 if you prefer.

Comment (evilnick) [reply docs/index.md]: I'm ambivalent, it's your show!

**Pull request merged**